util/pytest_log_filter.py

```diff
@@ -0,0 +1,16 @@
+import re
```

Comment:
If it's a python file that can be executed, as it seems to be the case, please add a shebang line, and mark the file executable.

Reply:
Does marking it excecutable actually transfer via Git?